data/Belize/Representatives/ep-popolo-v1.0.json

```diff
@@ -1,10 +1,6 @@
 {
   "posts": [
-    {
-      "id": "speaker",
-      "label": "Speaker",
-      "organization_id": "60da1355-8aef-4599-9292-5f796f99eb2a"
-    }
```

Comment:
We're losing this post, but I think that's acceptable: this isn't something we keep track of generally.